crates/madara/src/commands/orchestrator.rs

```diff
@@ -25,8 +39,17 @@ pub(crate) fn run(args_madara: MadaraRunnerConfigMode, shell: &Shell) -> anyhow:
     let args_pathfinder = PathfinderRunnerConfigMode::default().fill_values_with_prompt()?;
     commands::pathfinder::parse_params(&args_pathfinder)?;
 
+    // Check if the ATLANTIC_API was already set
+    let env_output = format!("{}/{}", ORCHESTRATOR_REPO_PATH, ORCHESTRATOR_ENV_FILE);
+    let _ = from_filename(env_output);
+    let prev_atlantic_api = env::var("MADARA_ORCHESTRATOR_ATLANTIC_API_KEY")
+        .unwrap_or_else(|_| "ATLANTIC_API_KEY".to_string());
```

Comment:
Why we fallback into "ATLANTIC_API_KEY"?

Reply:
We try to read the env file. If it's the first time, it won't be there, so the default value needs to be `ATLANTIC_API_KEY` to guide the user just a little bit more (see image below)
If the user selects Dummy prover, this won't be displayed at all

![Screenshot 2025-02-19 at 4 51 47 PM](https://github.com/user-attachments/assets/faa531c8-9689-4212-9e79-b7c758884657)